app/views/agencies/show.html.erb

```diff
@@ -5,11 +5,14 @@
 
 <span  class="page-header articleTitle">
   <h1 class="page-header articleTitle"><%= @agency.name %></h1>
-  <h4><%= @agency.street_address %></h4>
-  <h4><%= @agency.city %>, <%= State.find(@agency.state_id).name %> <%= @agency.zipcode %></h4>
```

Comment:
Hey, I know this has already been submitted, but can we turn this into a scope on `State` and use that in the view instead? Thanks.